package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@single-spa-foundry/cli",
+  "version": "0.0.0",
+  "description": "Deployment CLI for single-spa foundry",
+  "type": "module",
+  "main": "lib/js-api.js",
+  "bin": "lib/cli.js",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "cli": "cross-env NODE_OPTIONS=\"--loader ts-node/esm --experimental-specifier-resolution=node\" FOUNDRY_TOKEN=sample node src/cli.ts",
+    "format": "prettier --write .",
+    "check-format": "prettier --check .",
+    "lint": "eslint \"src/**/*\"",
+    "build": "rimraf lib && tsc",
+    "prepublishOnly": "pnpm run build",
```

Comment:
Is it common to use camel case for script commands?

Reply:
Yes. The `prepublishOnly` script is actually a special one that has a specific meaning with npm. See https://docs.npmjs.com/cli/v8/using-npm/scripts